spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -2757,6 +2757,20 @@
       "name": "spring.session.redis.flush-mode",
       "defaultValue": "on-save"
     },
+    {
+      "name": "spring.session.redis.repository",
+      "description": "Redis session repository implementation to use.",
+      "values": [
+        {
+          "value": "default",
```

Comment:
`RedisSessionRepository` by itself doesn't offer much inspiration on how to name this other than `default` or maybe `basic`.